releasenotes/config.yaml

```diff
@@ -33,7 +34,6 @@ sections:
   - [upgrade_visualization, Visualization Upgrade Notes]
   - [upgrade_misc, Misc. Upgrade Notes]
   - [deprecations, Deprecation Notes]
-  - [deprecations_algorithms, Algorithms Deprecations]
   - [deprecations_circuits, Circuits Deprecations]
   - [deprecations_primitives, Primitives Deprecations]
   - [deprecations_providers, Providers Deprecations]
```

Comment:
Let's add a `deprecations_c` while we're at it. We are unlikely to use it at first, but it just saves us from having to add it when we need it.